### ISSUE TITLE
cant do deauthentication method on 5 GHz band

### ISSUE THREAD

**Issue by genericc** [wifiphisher/wifiphisher#1094]
hi i want to try ask you can you help me with Alfa Network AWUS036ACH  adaptor?i have 2 same this adaptors.
im have try to deauth all clients from router on 5 ghz band but unsuccessfully  can you advice for me which software use for this 5ghz attack?and which command im must use on wifiphisher if this support on 5ghz band.
and how? thanks.

**Comment by sophron** [comment]
We do not support the 5ghz band as of now.

**Comment by genericc** [comment]
dear sophron thanks for your answer.please let me know here when you make deauth on 5 ghz band will
be wait, i have tested your deauth software on 2.4 band and have notice that your deauth software works much better like other software eg.aireplay-ng or mdk3 or mdk4. with your deauth software victim pc has disconnecting off right away.with other softwares its mitigate slowly and not realy effective.so if you create same on 5ghz bandi will be very happy.
ps.just 1 minus is when using your wifiphisher software its using very loot of pc cpu.its looks like when using software in same time going cryptocurrency mining process eg.mining monero:)
im realy sorry if im speak wrong but im noticed that because in 1.2 version when using cpu is fine 3% maybe using.on 1.4 version 100% :)

**Comment by sophron** [comment]
Unfortunately, we cannot provide an exact date. This is an open-source project that is run entirely by volunteers in their free time.

The CPU issue that you mention is interesting and I was not aware of that. I will have a look.

**Comment by genericc** [comment]
ok thanks for answer,

**Comment by chic01taliano** [comment]
As far as i understand it Wifiphisher utilizes mdk3 for deauthing so you can make use of that separately while utilizing Wifiphisher for deauthing the 2.4 band and create the rogue AP.

**Comment by bricco1981** [comment]
i'm not sure why people ask for 5ghz band? it's very simple! lanch wifiphisher on 2.4ghz, clone a 5 ghz network with your 2.4ghz usb adapter, with another adapter 5ghz, launch aireplay, or mdk on the network target an you ll make wifiphisher work on 5ghz band.
also if you clone a 5ghz network with a 2.4 ghz device you ll have benefit on longer distance, because 2.4ghz is more powerfull in term of distance.

**Comment by chic01taliano** [comment]
> i'm not sure why people ask for 5ghz band?

You need to DoS both bands in order to narrow user's choice. I think it can still be done succesfully via mdk4's; or at least that's what i use.

**Comment by dakrisht** [comment]
> i'm not sure why people ask for 5ghz band? it's very simple! lanch wifiphisher on 2.4ghz, clone a 5 ghz network with your 2.4ghz usb adapter, with another adapter 5ghz, launch aireplay, or mdk on the network target an you ll make wifiphisher work on 5ghz band.
> also if you clone a 5ghz network with a 2.4 ghz device you ll have benefit on longer distance, because 2.4ghz is more powerfull in term of distance.also that way you can clone the mac address of the acces point and use that mac on your 2.4ghz device. that way you have no problem with mdk or aireplay because your 5ghz card is working on another channel, on 5ghz band.

Yeah this is definitely the best workaround for deauthing 5Ghz bands. 2nd adapter and simply aireplay-ng that band and poof.

**Comment by chic01taliano** [comment]
Where i live routers have both bands and that means you need at least 3 adapters in order to overcome band steering.

**Comment by Commander-Crash** [comment]
Devs: Why Bother with 5ghz in 2020? 
Me: No one uses 2.4ghz in my area it's either 5ghz or wifi 6. But hey I digress.

**Comment by chic01taliano** [comment]
The only issue with 5ghz is that you need to be much closer to target.

**Comment by amalo201** [comment]
i believe that if you wanted you could use your AWUS036ACH adaptor with aircrack-ng to de-authenticate the clients connected in the 5Ghz band, and then if you have the luxury of owning a second external card like Alfa AWUS036NH you can use it with wifiphisher in combination with your laptops internal card to set up the AP and de-authenticate 2.4Ghz. a bit more complicated but its a workaround to get those 5ghz clients connected to your EAP.
